### PR TITLE
Update repo after initial code generation

### PR DESCRIPTION
**Pull request by tschaffter**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "data-node-client",
+    "version": "0.2.1",
+    "license": "Apache-2.0",
+    "devDependencies": {
+        "@openapitools/openapi-generator-cli": "^1.0.18-4.3.1"
+    },
+    "scripts": {
+        "generate:client": "openapi-generator generate -g python -o . package-name datanodeclient -i $npm_config_spec"
```
Comment (thomasyu888) [inline package.json:9]:
```suggestion
        "generate:client": "openapi-generator generate -g python -o . --package-name datanodeclient -i $npm_config_spec"
```

File: tox.ini [hunk 2]
```diff
@@ -1,9 +1,10 @@
 [tox]
-envlist = py27, py3
+envlist = py38
```
Comment (thomasyu888) [inline tox.ini:2]: Can we add py37?

File: .openapi-generator/FILES [hunk 3]
```diff
@@ -0,0 +1,104 @@
+.gitignore
```
Comment (thomasyu888) [inline .openapi-generator/FILES:1]: What are the purpose of these files?

File: tox.ini (hunk 2 above)
Comment (tschaffter) [reply tox.ini:2]: Sure

File: .openapi-generator/FILES (hunk 3 above)
Comment (tschaffter) [reply .openapi-generator/FILES:1]: Not 100% sure. The file *FILES* is a new addition of the openapi-generator version 5. I think that it's meant to replace .openapi-generator-ignore, i.e. files that are listed in *FILES* will be (re-)generated. To check.

**Review by thomasyu888 — APPROVED** [review]

**Pull request merged**